pegasus/sites.v3/code.org/public/alt-about/index.md.erb

```diff
@@ -196,8 +196,11 @@ Code.org is a member of the steering committee that helped establish the [K-12 C
 
 <hr/>
 
-<a href="http://www.guidestar.org/organizations/46-0858543/code-org.aspx" target="_blank">
-    <img src="https://widgets.guidestar.org/gximage2?o=9218725&l=v3" />
-</a> [![image](/images/fit-150/student_privacy_pledge.png)](http://studentprivacypledge.org/)
+<a href="https://www.guidestar.org/profile/46-0858543" target="_blank" rel="noopener noreferrer">
+  <object data="https://widgets.guidestar.org/gximage2?o=9218725&l=v4" type="image/png" style="vertical-align: middle; margin-right: 40px">
+    <img src="/images/guidestar-logo.png" />
+  </object>
+</a> 
```

Comment:
It might be good to factor this common content out into an embeddable view/partial.

Reply:
Which can also lend the advantage of using HAML instead of HTML for this snippet.